Backpack/Calendar/Classes/BPKCalendarCell.m

```diff
@@ -56,27 +54,29 @@ - (instancetype)initWithFrame:(CGRect)frame {
 - (void)layoutSubviews {
     [super layoutSubviews];
 
-    CGFloat padding = (CGRectGetWidth(self.bounds) - CGRectGetWidth(self.shapeLayer.bounds)) / 2.0 - 0.5;
-    if (padding > CGRectGetWidth(self.shapeLayer.bounds) / 8.0) {
-        padding = CGRectGetWidth(self.shapeLayer.bounds) / 8.0;
-    }
+    CGFloat TITLE_HEIGHT = 31;
+    CGFloat CIRCLE_HEIGHT = 31.6;
+    CGFloat sameDayOffset = 3.75;
+
+    CGFloat paddingX = (CGRectGetWidth(self.bounds) - CIRCLE_HEIGHT) / 2.0;
```

Comment:
I've reworked this as before it made assumptions about the background being the full height of the cell, and the label being placed centrally.
In the new price config, we'll need the circle and title to stay still while we add additional content below.